lrauv_ignition_plugins/example/example_comms_echo.cc

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+/*
+ * Development of this module has been funded by the Monterey Bay Aquarium
+ * Research Institute (MBARI) and the David and Lucile Packard Foundation
+ */
+
+/**
+ * This is a small example of usage of gazebo transport to send and receive
+ * data packets using the acoustic subsystem. For more info:
+ * https://github.com/osrf/lrauv/wiki/Examples-and-tutorials#direct-method
```

Comment:
@marcoag qq, will there a be instructions on how to use these examples in the wiki? If not, perhaps it's worth adding them here.

Reply:
There's a line that says how to run the example. Further documentation should be on the wiki, I guess it's better to have just one main source of documentation.